x-pack/plugin/esql/src/main/generated/org/elasticsearch/xpack/esql/expression/function/scalar/date/DayNameNanosEvaluator.java

```diff
@@ -71,14 +71,14 @@ public long baseRamBytesUsed() {
   public BytesRefBlock eval(int positionCount, LongBlock valBlock) {
     try(BytesRefBlock.Builder result = driverContext.blockFactory().newBytesRefBlockBuilder(positionCount)) {
       position: for (int p = 0; p < positionCount; p++) {
-        if (valBlock.isNull(p)) {
+        switch (valBlock.getValueCount(p)) {
+          case 0:
           result.appendNull();
```

Comment:
Could you indent here?

Reply:
Sure. I thought it was correct because spotless didn't change it. Only to remember now that spotless ignore these files :hehe: